### PR TITLE
fix(upstream) proper primary_key in kong schema

### LINKED ISSUE #2358: Cannot delete upstreams on Kong 0.10.1
### Summary

Hi,

I'm playing with upstreams on Kong 0.10.1 and I'm getting some strange errors while I'm trying to delete them.

### Steps To Reproduce

**# curl -X POST --url http://localhost:8001/upstreams/ --data "name=test"**
{"orderlist":[84,52,17,22,14,35,15,98,72,42,83,27,92,86,24,26,76,16,74,11,6,85,18,20,63,29,5,43,9,7,70,79,94,19,25,60,45,61,96,32,49,65,53,46,77,100,31,67,2,81,40,44,82,95,37,90,12,47,48,33,13,54,55,89,58,71,68,93,50,39,91,87,78,64,8,97,30,34,57,69,10,51,99,1,21,80,75,28,3,59,56,38,41,66,73,4,88,36,62,23],"slots":100,"id":"eac5ea6d-4b09-4447-8319-5816d0f8fcdf","name":"test","created_at":1491564431504}

**# curl --url http://localhost:8001/upstreams/test**
{"orderlist":[84,52,17,22,14,35,15,98,72,42,83,27,92,86,24,26,76,16,74,11,6,85,18,20,63,29,5,43,9,7,70,79,94,19,25,60,45,61,96,32,49,65,53,46,77,100,31,67,2,81,40,44,82,95,37,90,12,47,48,33,13,54,55,89,58,71,68,93,50,39,91,87,78,64,8,97,30,34,57,69,10,51,99,1,21,80,75,28,3,59,56,38,41,66,73,4,88,36,62,23],"slots":100,"id":"eac5ea6d-4b09-4447-8319-5816d0f8fcdf","name":"test","created_at":1491564431504}

**# curl -X DELETE --url http://localhost:8001/upstreams/test**
{"message":"An unexpected error occurred"}

### Additional Details & Logs

- Kong version 0.10.1
- Operating System CentOS Linux release 7.3.1611 (Core)
- Cassandra [cqlsh 5.0.1 | Cassandra 3.0.9 | CQL spec 3.4.0 | Native protocol v4]

and in the error.log I'm getting:

2017/04/07 11:27:24 [error] 4542#0: *324112 [lua] responses.lua:101: handler(): could not prepare query: [Invalid] Non PRIMARY KEY columns found in where clause: name , client: 127.0.0.1, server: kong_admin, request: "DELETE /upstreams/test HTTP/1.1", host: "localhost:8001"

I have some debug logs from other attempts:
2017/04/07 14:29:59 [debug] 20602#0: *2639 [lua] cluster.lua:602: prepare(): [lua-cassandra] preparing DELETE FROM upstreams WHERE name = ? AND id = ? on host
2017/04/07 14:29:59 [error] 20602#0: *2639 [lua] responses.lua:101: handler(): could not prepare query: [Invalid] Non PRIMARY KEY columns found in where clause: name , client: , server: kong_admin, request: "DELETE /upstreams/test HTTP/1.1", host: ":8001"

### PR DESCRIPTION
**Pull request by p0pr0ck5**
### Summary

DELETE against upstream objects is currently broken when using cassandra because the schema defines the primary key as `{id, name}` but both postgres and cassandra schemas actually define their PK with `id`. this causes failures because cassandra tries to delete against a non PK column.

### Full changelog

* correctly enumerate daos in upstream integration tests
* make Kong and DB schemas consistent

### Issues resolved

Fix #2358

### CODE REVIEW

File: spec/02-integration/03-admin_api/08-upstreams_routes_spec.lua [hunk 1]
```diff
@@ -37,25 +38,26 @@ dao_helpers.for_each_dao(function(kong_config)
 describe("Admin API", function()
   local client
   local config_db
```
Comment (thibaultcha) [inline spec/02-integration/03-admin_api/08-upstreams_routes_spec.lua]: I think this variable becomes unused now, and I am curious why the CI failed to catch that, if so.

Comment (thibaultcha) [inline spec/02-integration/03-admin_api/08-upstreams_routes_spec.lua]: (was it unused? and if so, did the linter caught it locally?)

Comment (p0pr0ck5) [inline spec/02-integration/03-admin_api/08-upstreams_routes_spec.lua]: It is unused, not sure why the linter didn't catch it. Didn't catch it locally either

Comment (thibaultcha) [inline spec/02-integration/03-admin_api/08-upstreams_routes_spec.lua]: Ok... Good to know. WE might want to double-check the luacheck settings under the `spec` folder then... 🤔

**Pull request merged**